test_suite/generic/test_generic.py

```diff
@@ -647,10 +655,11 @@ def test_cloud_cfg_netdev_rhel9(self, host):
 
 @pytest.mark.pub
 @pytest.mark.order(3)
+@pytest.mark.usefixtures('exclude_on_rhel_aws_strato')
 class TestsYum:
     # TODO: confirm if this test needs to be deprecated
     @pytest.mark.run_on(['rhel', 'fedora'])
-    def test_yum_repoinfo(self, host):
+    def test_yum_repoinfo(self, host, instance_data):
```

Comment:
Remove unused `instance_data` reference: 
```suggestion
    def test_yum_repoinfo(self, host):
```